dist/client.js

```diff
@@ -459,7 +459,6 @@
     define([], function() {
       return CrossStorageClient;
     });
-  } else {
-    root.CrossStorageClient = CrossStorageClient;
   }
+  root.CrossStorageClient = CrossStorageClient;
```

Comment:
That is a nice workaround, but I have a module loader system that loads some modules that actually need node require. This would get really ugly if I had to update the loader system to selectively do it.